src/odemis/gui/util/__init__.py

```diff
@@ -136,7 +136,15 @@ def get_home_folder():
     """ Return the home directory of the user running the Odemis GUI
     """
     # fall-back to HOME
-    folder = os.path.expanduser(u"~")
+    if sys.platform.startswith('linux'):
+        folder = os.path.expanduser(u"~")
+    elif sys.platform.startswith('win32'):
+        # Import functions here because wintypes in ctypes library cannot be opened in linux
```

Comment:
More importantly, add a comment that expanduser(u) fails with non-ascii usernames (in Python2), and add a link to the bug report for Python.
Also, it'd be simpler to say that `winknownpath` cannot be imported if not on Windows.